packages/core/src/types.ts

```diff
@@ -48,3 +50,26 @@ export interface SuggestionInput {
   arboristValues: IArboristNode[];
   latestPackages?: IDependencyMap;
 }
+
+export interface Plugin {
```

Comment:
For the future, we should make this an abstract class since people are instantiated new `Plugin`s